PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -8,55 +8,32 @@
 
 import Foundation
 
-// Example usage:
-//
-//    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(from: PennyPincherAndroidGesturesImporter.defaultAndroidFilePath ?? "") {
-//        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
-//            pennyPincherGestureRecognizer.templates.append(template)
-//        }
-//    }
-//
-
-public struct ImportedStrokes {
-    public let points: [CGPoint]
-    
-    public init(points: [CGPoint]) {
-        self.points = points
-    }
-}
-
-public struct ImportedGesture {
-    public let id: String
-    public let strokes: [ImportedStrokes]
-    
-    public init(id: String, strokes: [ImportedStrokes]) {
-        self.id = id
-        self.strokes = strokes
-    }
-    
-    public var allPoints: [CGPoint] {
-        return strokes.reduce([CGPoint]()) { points, stroke in
-            return points + stroke.points
-        }
-    }
-}
-
+/// PennyPincherAndroidGesturesImporter
+/// Example usage:
+///
+///    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(fromURL: androidGesturesFileURL) {
+///        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
+///            pennyPincherGestureRecognizer.templates.append(template)
+///        }
+///    }
+///
 public final class PennyPincherAndroidGesturesImporter {
     
-    public static var defaultAndroidFilePath: String? {
-        guard let path = Bundle.main.path(forResource: "gestures", ofType: "") else {
+    /// Default android gestures file is called 'gestures', no extension
+    public static var defaultAndroidFileURL: URL? {
```

Comment:
"default Android file URL" sounds like this file is about Android, not about Android-type gestures. Maybe omit Android from this symbol, and leave it to be communicated by the type prefix: `PennyPincherAndroidGesturesImporter.defaultFileURL` or `.defaultImportFileURL`